server/channels/api4/bot_test.go

```diff
@@ -183,16 +197,20 @@ func TestPatchBot(t *testing.T) {
 		})
 		require.NoError(t, err)
 		CheckCreatedStatus(t, resp)
-		defer th.App.PermanentDeleteBot(th.Context, createdBot.UserId)
+		defer func() {
+			appErr := th.App.PermanentDeleteBot(th.Context, createdBot.UserId)
+			assert.Nil(t, appErr)
+		}()
 
 		th.TestForSystemAdminAndLocal(t, func(t *testing.T, client *model.Client4) {
 			botPatch := &model.BotPatch{
 				Username:    sToP(GenerateTestUsername()),
 				DisplayName: sToP("an updated bot"),
 				Description: sToP("updated bot"),
 			}
-			patchedBot, patchResp, err2 := client.PatchBot(context.Background(), createdBot.UserId, botPatch)
-			require.NoError(t, err2)
+			_, _, err = client.PatchBot(context.Background(), createdBot.UserId, botPatch)
+			require.NoError(t, err)
+			patchedBot, patchResp, _ := client.PatchBot(context.Background(), createdBot.UserId, botPatch)
```

Comment:
I wonder why you are changing this piece of code. The error gets properly checked. What is the reason for a change?

Reply:
Because when I keep it as err:= it shows shadow variable in goci lint as err has been already declared in that test function.

Reply:
And I can't give it different variable name. As the convention is to use err for type error. Before it was declared as appErr.

Reply:
We can keep the existing code, which uses `err2` as a variable name. It's not pretty, but let's not change more than necessary as part of this PR.